### PR TITLE
Dev-Apps-923 - Enable download in uv

### PR DESCRIPTION
**Pull request by darrowcousc**
*** deployment to dev for testing ***

Connected to [APPS-923](https://jira.library.ucla.edu/browse/APPS-923)

**Aceptence Criteria**

- [x] User can download images using the download button in universal viewer
- [x] User can specify size of download

**Screen shot**
image

**Changed files**
- config/uv/uv-config.json

### CODE REVIEW

File: src/uv-config.json [hunk 1]
```diff
@@ -3,12 +3,40 @@
...
+            "printEnabled": true,
+            "shareEnabled": false
+          },
+          "content": {
+            "bookmark": "Add to bookmarks",
+            "download": "Download ",
+            "embed": "Embed",
+            "feedback": "Feedback",
+            "exitFullScreen": "",
+            "fullScreen": "",
```
Comment (sourcefilter) [inline src/uv-config.json]: Why are we removing the tooltip labels for 'fullScreen' and 'exitFullScreen'? And for that matter, do we need to include any of these labels? I'd think the UV defaults would be fine.

File: src/uv-config.json [hunk 2]
```diff
@@ -3,12 +3,40 @@
         "rightPanelEnabled": false
     },
     "modules": {
-        "footerPanel": {
-            "options": {
-                "downloadEnabled": false,
-                "fullscreenEnabled": true,
-                "shareEnabled": false
-            }
+      "headerPanel": {
```
Comment (sourcefilter) [inline src/uv-config.json:6]: Do we need this section? It looks like this is the UV default.

**Review by sourcefilter — CHANGES_REQUESTED** [review]
I added a few specific comments, but in general I'm not clear why any changes are needed beyond changing "downloadEnabled" to true and adding "printEnabled": true in the modules/FooterPanel/options section.

Could you either remove the other changes or add a note explaining their purpose?

[UPDATE] we also want to keep the dialogoue options, I imagine

File: src/uv-config.json (hunk 2 above)
Comment (darrowcousc) [reply src/uv-config.json:6]: I removed the optional code. The lines `moreInfoEnabled` and `shareEnabled` default to `true` so setting them `false` needs to be retained.

**Review by sourcefilter — APPROVED** [review]

**Pull request merged**